packages/sirius-web/backend/sirius-web-table/src/main/java/org/eclipse/sirius/web/table/services/ForkedStudioJdbcClient.java

```diff
@@ -0,0 +1,115 @@
+/*******************************************************************************
+ * Copyright (c) 2024 CEA LIST.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.sirius.web.table.services;
+
+import org.eclipse.sirius.components.core.api.IRepresentationInput;
+import org.eclipse.sirius.web.domain.boundedcontexts.project.Project;
+import org.eclipse.sirius.web.domain.boundedcontexts.project.services.api.IProjectCreationService;
+import org.eclipse.sirius.web.domain.boundedcontexts.representationdata.services.api.IRepresentationContentSearchService;
+import org.eclipse.sirius.web.domain.boundedcontexts.representationdata.services.api.IRepresentationContentUpdateService;
+import org.eclipse.sirius.web.domain.services.Success;
+import org.springframework.jdbc.core.simple.JdbcClient;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * Implementation IForkedStudioJdbcServices used to update a forked studio.
+ *
+ * @author mcharfadi
+ */
+@Service
+@Transactional
+public class ForkedStudioJdbcClient implements IForkedStudioJdbcServices {
+
+    public static final String STUDIO_NATURE = "siriusComponents://nature?kind=studio";
+
+    private final JdbcClient jdbcClient;
+
+    private final IRepresentationContentUpdateService representationContentUpdateService;
+
+    private final IRepresentationContentSearchService representationContentSearchService;
+
+    private final IProjectCreationService projectCreationService;
+
+    public ForkedStudioJdbcClient(JdbcClient jdbcClient, IRepresentationContentUpdateService representationContentUpdateService, IRepresentationContentSearchService representationContentSearchService, IProjectCreationService projectCreationService) {
+        this.jdbcClient = jdbcClient;
+        this.representationContentUpdateService = representationContentUpdateService;
+        this.representationContentSearchService = representationContentSearchService;
+        this.projectCreationService = projectCreationService;
+    }
+
+    @Override
+    public void insertViewDocument(UUID documentId, UUID semanticProjectId, String content, String name) {
+        String sql = "INSERT INTO document (id, semantic_data_id, name, content, created_on, last_modified_on) VALUES (:id, :semantic_data_id, :name, :content, :created_on, :last_modified_on)";
```

Comment:
Don't hesitate to use multiline string to make those kind of queries more readable in the future.

Reply:
I had issues executing queries with multiline string so I did not change this file.